src/styles/main.scss

```
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+@import "./blocks.scss";
+@import "./variables.scss";
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

```suggestion
  align-items: center;
  height: 100vh;
```
for show in the middle of the screen container should have height on full height of screen